fe/src/main/java/org/apache/doris/alter/MaterializedViewHandler.java

```diff
@@ -85,6 +88,54 @@ public MaterializedViewHandler() {
         super("materialized view");
     }
 
+
+    // for batch submit rollup job, tableId -> jobId
+    // keep table's not final state job size. The job size determine's table's state, = 0 means table is normal,otherwrise is rollup
```

Comment:
```suggestion
    // keep table's not final state job size. The job size determine's table's state, = 0 means table is normal, otherwise is rollup
```

Reply:
👌